tests/mysql/test-mysql.h

```diff
@@ -112,6 +112,13 @@ class test_context : public test_context_base
         // SQL mode to avoid silent truncation of too long values.
         return sql_mode.find("STRICT_") == std::string::npos;
     }
+
+    virtual bool has_trailing_spaces_in_char_type(session& sql) const 
+    {
+        //turn on padding on mysql - trimming by default
+        sql << "SET @@session.sql_mode = 'PAD_CHAR_TO_FULL_LENGTH'";
```

Comment:
Ok. I'll catch for exception here with message indicating support as of MySQL 5.1.20.